include/xtensor/xstrided_view.hpp

```diff
@@ -628,7 +630,7 @@ namespace xt
     template <class CT, class S, class CD>
     inline auto xstrided_view<CT, S, CD>::data_xbegin() const noexcept -> const_container_iterator
     {
-        return data_xbegin_impl(m_data.begin());
+        return data_xbegin_impl(m_data.cbegin());
```

Comment:
This should not be necessary and this will mask a bug that we probably have in xtensor (see #730).

Reply:
👎 the policy so far was to use `cbegin` / `cend` versions whenever possible

Reply:
So my "at least until we understand the bug". I just want to be sure we don't miss anything.

Reply:
(as discussed in person)

Reply:
I agree with @JohanMabille's "at least until we understand the bug".